src/pages/collection/series/TmdbLinking.tsx

```diff
@@ -179,7 +179,7 @@ const TmdbLinking = () => {
           TmdbShowID: tmdbId,
           TmdbEpisodeID: overrideId,
           Index: index,
-          Rating: (overrideId === 0 ? MatchRatingType.None : MatchRatingType.UserVerified),
+          Rating: MatchRatingType.UserVerified,
```

Comment:
Why? Why do you think it existed in the first place and why did you remove it?

Reply:
It is no longer needed. In fact, it's getting in the way now. Because the server now distinguished between auto-matched (AKA "none") and overridden (AKA "user verified") non-links, where the override will prevent the automagic linking from changing the link, while the auto-matched will not. This change is just to brink the UI in line with what the server will return once the match is finalized.